core/lib/Thelia/Controller/Admin/ModuleController.php

```diff
@@ -406,4 +407,64 @@ public function informationAction($module_id)
 
         return new JsonResponse([ 'title' => $title, 'body' => $content], $status);
     }
+
+    public function documentationAction($module_id)
+    {
+        if (null !== $response = $this->checkAuth(AdminResources::MODULE, array(), AccessManager::VIEW)) {
+            return $response;
+        }
+
+        $status = 200;
+
+        $content = null;
+
+        if (null !== $module = ModuleQuery::create()->findPk($module_id)) {
+            $title = $module->setLocale($this->getSession()->getLang()->getLocale())->getTitle();
+
+            $finder = null;
+
+            // Check if the module has declared a documentation
+            $moduleDescriptor = $module->getAbsoluteConfigPath() . DS . 'module.xml';
+
+            if (false !== $xmlData = @simplexml_load_file($moduleDescriptor)) {
+                $documentationDirectory = (string) $xmlData->documentation;
+
+                if (! empty($documentationDirectory)) {
+                    $finder = Finder::create()->files()->in($module->getAbsoluteBaseDir())->name('*.md');
```

Comment:
I think that here too, the name shouldn't be case sensitive, I'd really like to call my doc ```README.MD```.
I think ```"/.+\.md$/i"``` would be better